macros/models/_sector/erc4337/erc4337_userops_basics.sql

```diff
@@ -20,21 +21,22 @@ with userop as (
         , actualGasCost/1e18 as op_fee
     from {{ userops_evt_model }}
     {% if is_incremental() %}
-        WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
     {% endif %}
 )
 , handleops as (
     select call_tx_hash as tx_hash,
            beneficiary
     from {{ handleops_call_model }}
     {% if is_incremental() %}
-        WHERE call_block_time >= date_trunc("day", now() - interval '1 week')
+    WHERE call_block_time >= date_trunc("day", now() - interval '1 week')
     {% endif %}
 )
 
 select
       userop.blockchain
     , userop.version
+    , userop.block_month
```

Comment:
pull `block_month` out of macro, to be added to query and built in table, then partition